src/quo2/components/wallet/progress_bar/style.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.wallet.progress-bar.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn- border-and-background-color
```

Comment:
Maybe rename this function to something like `color-props`?

Reply:
I feel the name `color-styles` is more of a generic name. While `border-and-background-color` is more explicit about what the method will return.